### PR TITLE
[3984] Add _Since_ and _Last Updated_ tags to all pages

### PR DESCRIPTION
**Pull request by jvega190**
- Add styles to since and last updated tags

https://github.com/craftercms/craftercms/issues/3984

### CODE REVIEW

**Review by rart — APPROVED** [review]

File: ui/craftercms-sphinx-theme/src/sphinx/layout.html [hunk 1]
```diff
@@ -20,6 +20,30 @@
 {% set upToDate = meta.get(upToDate_key) %}
 {% endif %}
 
+{% set sinceLabel = '' %}
+{% set sinceLabel_key = 'since-label' %}
+{%- if check_meta and sinceLabel_key in meta %}
+{%- set sinceLabel = meta.get(sinceLabel_key) %}
```
Comment (alhambrav) [inline ui/craftercms-sphinx-theme/src/sphinx/layout.html:26]: Indent?

File: ui/craftercms-sphinx-theme/src/sphinx/layout.html [hunk 2]
```diff
@@ -20,6 +20,30 @@
...
+{%- endif %}
+
+{% set sinceVersion = '' %}
+{% set sinceVersion_key = 'since-version' %}
+{%- if check_meta and sinceVersion_key in meta %}
+{%- set sinceVersion = meta.get(sinceVersion_key) %}
```
Comment (alhambrav) [inline ui/craftercms-sphinx-theme/src/sphinx/layout.html:32]: Indent?

File: ui/craftercms-sphinx-theme/src/sphinx/layout.html [hunk 3]
```diff
@@ -20,6 +20,30 @@
...
+{%- endif %}
+
+{% set updatedOnLabel = '' %}
+{% set updatedOnLabel_key = 'updated-label' %}
+{%- if check_meta and updatedOnLabel_key in meta %}
+{%- set updatedOnLabel = meta.get(updatedOnLabel_key) %}
```
Comment (alhambrav) [inline ui/craftercms-sphinx-theme/src/sphinx/layout.html:38]: Indent?

File: ui/craftercms-sphinx-theme/src/sphinx/layout.html [hunk 4]
```diff
@@ -20,6 +20,30 @@
...
+{%- endif %}
+
+{% set lastUpdatedVersion = '' %}
+{% set lastUpdatedVersion_key = 'last-updated' %}
+{%- if check_meta and lastUpdatedVersion_key in meta %}
+{%- set lastUpdatedVersion = meta.get(lastUpdatedVersion_key) %}
```
Comment (alhambrav) [inline ui/craftercms-sphinx-theme/src/sphinx/layout.html:44]: Indent?

**Review by alhambrav — CHANGES_REQUESTED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**